CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 -  **Breaking:** The `kubectl` configuration file can now be fully-specified using `config_output_path`. Previously it was assumed that `config_output_path` referred to a directory and always ended with a forward slash. This is a breaking change if `config_output_path` does **not** end with a forward slash (which was advised against by the documentation).
 - Changed logic for setting default ebs_optimized to only require maintaining a list of instance types that don't support it (by @jeffmhastings)
 - Bumped minimum terraform version to 0.12.2 to prevent an error on yamlencode function (by @toadjaune)
+- Added support for Windows workers AMIs (by @hodduc)
```

Comment:
Plus... this is not a change, but it's a new feature. Please move this to the **Added** section.

Reply:
OK. I moved it.